students/saleta_sebastian/lesson_03_functions/exponential.py

```diff
@@ -5,9 +5,7 @@ def power(a, n):
         return a * power(a, n - 1)
 
 
-print(power(float(input()), int(input())))
-
-
-a = float(input())
-n = int(input())
-power(a, n)
+if __name__ == '__main__':
+    var1 = float(input())
```

Comment:
Getting input from the user could be moved to main() function same as line 11. So when the module will be executed as a script run main() function which you will add in line 9. You will have also opportunity to run main() in case import.

Please introduce this change in rest of your scripts.